### PR TITLE
Plushed With Greatness

### PR DESCRIPTION
**Pull request by dustylens**
## About the PR

Adds 5 new plushies from Ghostprince. 

They can be found in the cuddly critter vend and in the prize lathe pool.

They cannot be found in loadouts because:

## Why / Balance

Weh

## How to test

Buy plushies. Have a tea party. 

## Media

ss+(2024-11-23+at+09 29 18)

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

🆑 
-add: Added new plushies from Ghost Prince.

### CODE REVIEW

File: Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/cuddlycritter.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@
     PlushieHampter: 3
     PlushieLizard: 3
     PlushieSpaceLizard: 3
+    Plushie_Botanist: 3
+    Plushie_Cmo: 3
+    Plushie_Construction: 3
+    Plushie_Engineer: 3
+    Plushie_SharkmMinnow: 3
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/cuddlycritter.yml:12]: Per naming please swap to NFPlushieNAME format

File: Resources/Prototypes/_NF/Entities/Structures/Machines/lathe.yml [hunk 2]
```diff
@@ -1014,6 +1014,11 @@
       - PlushieBeeRecipe
       - PlushieLizardRecipe
       - PlushieSpaceLizardRecipe
+      - Plushie_BotanistRecipe
+      - Plushie_CmoRecipe
+      - Plushie_EngineerRecipe
+      - Plushie_ConstructionRecipe
+      - Plushie_SharkmMinnowRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Structures/Machines/lathe.yml:1021]: Will say that these should probably be in the emag side of things if only for consistency with the other plushies (special callouts to active players seems a bit gauche if it isn't fairly rare or obscure).

File: Resources/Prototypes/_NF/Recipes/Lathes/prizecounter.yml [hunk 3]
```diff
@@ -66,6 +66,31 @@
...
+
+- type: latheRecipe
+  id: Plushie_EngineerRecipe
+  result: Plushie_Engineer
+  parent: BasePrize15Recipe
+
+- type: latheRecipe
+  id: Plushie_SharkmMinnowRecipe
+  result: Plushie_SharkmMinnow
+  parent: BasePrize15Recipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/prizecounter.yml:92]: Per the other emag comment, recommending to treat this in kind - move to the bottom with the other EMAG contents, change price to 50.

File: Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml [hunk 4]
```diff
@@ -473,6 +513,25 @@
     sprite: _NF/Objects/Fun/catplushie.rsi
     state: white
 
+- type: entity
+  parent: PlushieCarp
+  id: Plushie_SharkmMinnow
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml:518]: Typo in your name here.

File: Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml [hunk 5]
```diff
@@ -23,6 +23,46 @@
       path: /Audio/_NF/Items/Toys/wehhelp.ogg
   - type: StinkyTrait
 
+- type: entity
+  parent: PlushieSpaceLizard
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml:27]: Unless you want muffled wehs, should just parent off of the base lizard plush.

Comment (whatston3) [reply Resources/Prototypes/_NF/Entities/Objects/Fun/toys.yml:27]: Also, bonus points if you fiddle with the audio params to get different or varying pitches (`pitch`, `variation`)

**Review by whatston3 — COMMENTED** [review]
Looks good, got a few minor recommendations, would like it if they were in the plushie mail listing as a rarity.

I do think there's something to be said for funny easter eggs being rare or tricky to get.

**Review by dvir001 — APPROVED** [review]

**Comment by dvir001** [comment]
image

**Pull request merged**